### ISSUE TITLE
Content request boxes on upcoming pages

### ISSUE THREAD

**Issue by rossettistone** [rossettistone/bizport#142]
Add pages that don't have the content yet and under content write "coming soon" or have a box "What content would you like here?" Submit button

**Comment by rossettistone** [comment]
Thinking this through to build a spec:

I'm thinking of a snippet in the CMS (e.g. `{{ cms::partial::shared/feedback_form:networking }}`) that could be added to any page to insert a text field and submit button. The form data would then be visible on a tab in the admin panel at `bizport.longbeach.gov/admin`

Would this be a good approach?

**Comment by rossettistone** [comment]
@lisaratner ^^

**Comment by lisaratner** [comment]
@rossettistone sounds good!

**Comment by rossettistone** [comment]
So I think when we closed this a few days ago during the planning meeting, our reason was that we figured the "use Intercom to suggest content" method that we added on those empty pages was sufficient, and we weren't going to spend the effort to build a more complex feedback system. My bad for not putting that into the story when we closed it.

Does that reason still stand, or should we reopen this issue and discuss what work remains to be done?
